### DISCUSSION TITLE
Fields Select with multiple

### DISCUSSION THREAD

**Discussion by incoming-th** [filamentphp/filament#508] [Ideas]
Hi,

Would it be possible - if not already - to have the **Select** field compatible with multiple? Same as Selectize.js for example:

https://selectize.dev/demos/2016/01/01/max-items/

That will allow to select one or more items. Apologize if this is already covered, but I did not find it in the doc.

Thanks

**Comment by danharrin** [comment]
Hi, this is not officially supported as its not documented, but you can find an internal `MultiSelect` component that we use in the `UserResource` and play around with it.

**Reply by incoming-th** [reply]
This discussion can be closed now as it is now supported.
